components/blitz/resources/omero/api/RenderingEngine.ice

```diff
@@ -20,57 +20,488 @@ module omero {
+                 */
                 idempotent int getDefaultT() throws ServerError;
+
+                /**
+                 * Sets the index of the default focal section. This index is
+                 * used to define a default plane.
+                 *
+                 * @param z The value to set.
+                 */
                 idempotent void setDefaultZ(int z) throws ServerError;
+
+                /**
+                 * Sets the default timepoint index. This index is used to
+                 * define a default plane.
+                 *
+                 * @param t The value to set.
+                 */
                 idempotent void setDefaultT(int t) throws ServerError;
+
+                /**
+                 * Returns the {@link omero.model.Pixels} set the Rendering
+                 * engine is for.
+                 */
                 idempotent omero::model::Pixels getPixels() throws ServerError;
+
+                /**
+                 * Returns the list of color models supported by the Rendering
+                 * engine.
+                 */
                 idempotent IObjectList getAvailableModels() throws ServerError;
+
+                /**
+                 * Returns the list of mapping families supported by the
+                 * Rendering engine.
+                 */
                 idempotent IObjectList getAvailableFamilies() throws ServerError;
+
+                /**
+                 * Sets the quantization strategy. The strategy is common to
+                 * all channels.
+                 *
+                 * @param bitResolution The bit resolution defining associated
+                 *                      to the strategy.
+                 */
                 idempotent void setQuantumStrategy(int bitResolution) throws ServerError;
+
+                /**
+                 * Sets the sub-interval of the device space i.e. a discrete
+                 * sub-interval of \[0, 255].
+                 *
+                 * @param start The lower bound of the interval.
+                 * @param end The upper bound of the interval.
+                 */
                 idempotent void setCodomainInterval(int start, int end) throws ServerError;
+
+                /**
+                 * Returns the quantization object.
+                 */
                 idempotent omero::model::QuantumDef getQuantumDef() throws ServerError;
+
+                /**
+                 * Sets the quantization map, one per channel.
+                 *
+                 * @param w  The channel index.
+                 * @param family The mapping family.
+                 * @param coefficient The coefficient identifying a curve in
+                 *                    the family.
+                 * @param noiseReduction Pass <code>true</code> to turn the
+                 *                       noise reduction algorithm on,
+                 *                       <code>false</code> otherwise.
```

Comment:
I don't think these need fixing now - as it was in the original docs - but this use of 'otherwise' for a parameter is not quite right. It's fine for a return value which is dependent on something but here it is simply 'The value passed to the noise reduction algorithm' or 'The boolean value...'  This occurs a good number of times in this file where booleans are passed in.